scripts/main.js

```diff
@@ -1,5 +1,5 @@
 $.ajax({
-    url: 'http://namegame.willowtreemobile.com:2000',
+    url: 'https://willowtreeapps.com/api/v1.0/profiles/',
```

Comment:
There seems to be missing people in this endpoint (unless that was done on purpose):

e.g. https://willowtreeapps.com/api/v1.0/profiles?limit=50&skip=87

<img width="515" alt="image" src="https://cloud.githubusercontent.com/assets/1592403/22444989/5accaf5a-e713-11e6-8765-4589e59c5ba8.png">

Also the [related endpoint on `willowtreeapps.com`](https://github.com/willowtreeapps/willowtreeapps.com/blob/fca86fc578d031da9b52992388996abfa31a5e28/app/server/routes/api.js#L277) led me to [Contentful's API on Entries](https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/search-parameters). As `willowtreeapps.com`'s endpoint passes through the query parameters given to Contentful, all of the search parameters Contentful provides are also available for the candidate to use.

Maybe consider documenting some of the query parameters in this repo. Or, just linking straight to Contentful's documentation on the available query parameters could be an option (with an example, e.g. the `?skip` query above).

It would be neat to include because it's just another thing that candidates can use in their test project if they wish, and it would get them into a API spec vs. just using an endpoint that returns static data (as the old namegame endpoint did).